onnxruntime/test/contrib_ops/matmul_4bits_test.cc

```diff
@@ -342,6 +342,14 @@ void TestMatMulNBitsTyped(std::optional<float> abs_error = std::nullopt,
     RunTest<AType>(opts);
   }
 #endif  // !defined(USE_DML) && !defined(USE_WEBGPU)
+#if defined(USE_WEBGPU)
```

Comment:
Also enable the bias test for matmul_8bits_tests.cc?

Reply:
done